src/app/find-help/material-support-search/material-support-search-form/material-support-search-form.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, Output, EventEmitter } from '@angular/core';
+import { MaterialAidOffer } from '../../../../api/model/materialAidOffer';
+
+const categories = Object.entries(MaterialAidOffer.CategoryEnum).map(([key, value]) => ({
+  code: key,
+  label: value,
+}));
+
+export interface MaterialSupportQuery {
+  location?: {
+    city: string;
+    region: string;
+  };
+  category?: string;
+}
+interface Option {
+  code: string;
+  label: string;
+}
+@Component({
+  selector: 'app-material-support-search-form',
+  templateUrl: './material-support-search-form.component.html',
+  styleUrls: ['./material-support-search-form.component.scss'],
+})
+export class MaterialSupportSearchFormComponent {
```

Comment:
Please name all the files, folders, types, classes, etc. using `material-aid`, not `material-support`, for consistency with Open API (services, endpoints, types are all named like that).